src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
```

Comment:
@danmosemsft Some people may disagree, but if there's conflict between "what Desktop does" and "what Mono does", Mono is **wrong**.  _Especially_ for security features.  If someone wrote code expecting CAS to work on full framework and has it running on Core via the retargeting shims then their Deny or PermitOnly **absolutely must not** "do nothing".

And while I do see that Desktop can throw a NotSupportedException from Deny, it has no such obvious exception from PermitOnly.

So, we can have precedent for NSE in Deny(); but we need PNSE in PermitOnly().